README.rst

```diff
@@ -0,0 +1,254 @@
+========================================================
+ REANA example - ALICE simple analysis demonstrator
+========================================================
+
+.. image:: https://img.shields.io/travis/reanahub/reana-demo-alice-pt-analysis.svg
+   :target: https://travis-ci.org/reanahub/reana-demo-alice-pt-analysis
+
+.. image:: https://badges.gitter.im/Join%20Chat.svg
+   :target: https://gitter.im/reanahub/reana?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge
+
+.. image:: https://img.shields.io/github/license/reanahub/reana-demo-alice-pt-analysis.svg
+   :target: https://raw.githubusercontent.com/reanahub/reana-demo-alice-pt-analysis/master/LICENSE
+
+About
+======
+
+This demo takes an `analysis example <http://opendata.cern.ch/record/1200>`_ from the CERN Open Data portal and illustrates
```

Comment:
Better invert the logic, first explain the example and then say from where it was taken:

```
This REANA reproducible analysis example demonstrates the use of the ALICE analysis framework to facilitate the analysis of ALICE collision and simulated data samples for both pp and PbPb data taken in the year 2010.  The example was taken from ...
```